src/main/java/core/basesyntax/service/handlers/OperationHandler.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service.handlers;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Transaction;
+
+public interface OperationHandler {
+    void processOperation(Transaction.Operation operation, Fruit fruit, Integer quantity);
```

Comment:
```suggestion
    void  handle(Fruit fruit, Integer quantity);
```

Reply:
great comment, I didn't even mention, that I had a redundant parameter here, thank you.